app/Console/Commands/SendPublishedAnnouncementEmails.php

```
@@ -0,0 +1,129 @@
+                    return self::FAILURE;
+                }
+
+                $delivery = AnnouncementEmailDelivery::query()->find((int) $retryDeliveryId);
+
+                if (! $delivery || ! $deliveryService->retryFailedDelivery($delivery)) {
+                    $this->error('Only an existing terminally failed delivery can be retried.');
+
+                    return self::FAILURE;
+                }
+
+                $result = $deliveryService->processDelivery($delivery->fresh());
```

PHPStan (2.2.8)

[warning] Parameter `#1` $delivery of method App\Services\AnnouncementEmailDeliveryService::processDelivery() expects App\Models\AnnouncementEmailDelivery, App\Models\AnnouncementEmailDelivery|null given.


(argument.type)